docs/docs/publishing/cli.md

```diff
@@ -16,26 +16,26 @@ Requires [Node.js](https://nodejs.org/en/download/) version 14.17 or above.
 
 ## Commands
 
-### `rune start`
+### `rune create`
 
-Starts a development server that emulates your game running in the Rune app. This interface is identical to the one described in [Simulating Multiplayer](simulating-multiplayer.md).
+Creates a new example game using Vite template.
 
 ```bash
-cd game/path && rune start
+rune creae
```

Comment:
typo

Reply:
Fixed in https://github.com/rune/rune-games-sdk/pull/200/commits/a04168c5b7e21fbdbcca8b3a178b795cca997db4